README.md

```diff
@@ -5,7 +5,13 @@
 
 Polling feature for Mattermost's custom slash command.
 
-## Setup Mattermost
+## Requirements
+- [go](https://golang.org/)
+    - Verion 1.7 or 1.8
+- [Mattermost server](https://about.mattermost.com/)
+    - Version 3.0 or newer
```

Comment:
If we merge #37, Mattermost must have endpoints for API version4.
The early beta version of Mattermost API v4 released at [Mattemost v3.7](https://about.mattermost.com/mattermost-3-7/#api-v4), so `version 3.7 or newer` is good.